core/templates/dev/head/pages/exploration-player-page/services/extract-image-filenames-from-state.service.ts

```diff
@@ -131,11 +131,11 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
         // But it actually contains the filename only. We use the variable
         // filename instead of filepath since in the end we are retrieving the
         // filenames in the exploration.
-        var filename = JSON.parse(
-          imageTagList[i].getAttribute('filepath-with-value'));
-        filenames.push(filename);
+        var imageId = JSON.parse(
```

Comment:
Please update the above comment accordingly.

Reply:
done